### ISSUE TITLE
Start building out SvelteKit structure on a branch

### ISSUE THREAD

**Issue by eyeseast** [MuckRock/documentcloud-frontend#388]
SvelteKit's [project structure](https://kit.svelte.dev/docs/project-structure) has surprisingly little overlap with our current code layout. This means we _should_ be able to start stubbing out routes and data flows on a branch without having to migrate the entire codebase. We can work on SvelteKit while still fixing bugs in the current structure.

Everything should be able to live on a `sveltekit` branch that we can merge into as needed.

```
my-project/
├ src/
│ ├ lib/
│ │ ├ server/
│ │ │ └ [your server-only lib files]
│ │ └ [your lib files]
│ ├ params/
│ │ └ [your param matchers]
│ ├ routes/
│ │ └ [your routes]
│ ├ app.html
│ ├ error.html
│ ├ hooks.client.js
│ ├ hooks.server.js
│ └ service-worker.js
├ static/
│ └ [your static assets]
├ tests/
│ └ [your tests]
├ package.json
├ svelte.config.js
├ tsconfig.json
└ vite.config.js
```

We need:

- `routes`
- `lib`
- `vite.config.js`
- SvelteKit things added to `svelte.config.js`
- Update `package.json` with Vite and SvelteKit dependencies, dropping webpack stuff
- `netlify.toml` so we can have a working branch
- `app.html` and `error.html`

Once all of that is in place, we can decide how to move components into SvelteKit.

**Comment by eyeseast** [comment]
Got the initial pieces of this going on the `sveltekit` branch. It looks like anything that imports the router, which is almost everything, is going to break. That means we need to keep focus on unwinding that dependency as we work on other things.

**Comment by eyeseast** [comment]
Basic structure is in place and deployed here: https://sveltekit.muckcloud.com/home/. Only homepage and flat pages work right now, but routes are stubbed out for the viewer and manager.
